src/graph_compute/update_function.h

```diff
@@ -58,7 +65,16 @@ typedef struct vertex_t vertex_t;
 //  a sched_t datatype which includes whatever per-vertex data
 //  they need to perform their scheduling (e.g., priority, satisfied, dependencies etc.)
 #if PAGERANK
-  inline static void fillInNodeData(vertex_t * nodes, const vid_t cntNodes) {
+  inline static uint64_t hashOfVertexData(data_t * vertex) {
+    uint64_t result = 0;
+    memcpy(static_cast<void *>(&result),
```

Comment:
I just want to copy the bits from the app-specific data into a uint64_t and xor them across all nodes.  What should I do instead?